### PR TITLE
Advice, self esteem topic

### PR DESCRIPTION
**Pull request by SLVRFOX**
Monika asks about the players self esteem and gives some advice.
Hopefully I didn't mess anything up. If I did tell me and I'll clean up!

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -138,6 +138,41 @@ label monika_bad_day:
...
     return
 
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['depression','self esteem','self-esteem','self image','self-image']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_self-image')
+    monika_random_topics.append('monika_self-image')
+
+label monika_self-image
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Label names cannot have dashes `-`. Please remove the dash.

**Comment by SLVRFOX** [comment]
"-" removed

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -138,6 +138,41 @@ label monika_bad_day:
...
     return
 
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['depression','self esteem','self-esteem','self image','self-image']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_selfimage')
+    monika_random_topics.append('monika_selfimage')
+
+label monika_self-image
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: You forgot to change the label itself.

**Comment by SLVRFOX** [comment]
I fixed that one too. Sorry!

**Comment by monikLover** [comment]
Reviewed and edited. Approved.

**Pull request merged**